build-in-container/scripts/setup.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+echo "------------ Setting up Mariner Build Environment ------------"
+
+# build variables
+BUILD_DIR="/tmp/mariner/build"
+CCACHE_DIR="/tmp/mariner/ccache"
+CBL_MARINER_GIT_URL="https://github.com/microsoft/CBL-Mariner.git"
+CHROOT_DIR="/temp/DockerStage/"
+DISABLE_UPSTREAM_REPOS="n"
+IMAGE_CONFIG_DIR="/sources/scripts/toolkit/imageconfigs"
+MARINER_RELEASE_TAG="2.0-stable"
+LOG_LEVEL="info"
+OUT_DIR="/tmp/mariner/out"
+RUN_CHECK="y"
+SOURCE_URL="https://cblmarinerstorage.blob.core.windows.net/sources/core"
+SPECS_DIR="/sources/SPECS"
+
+download_mariner_toolkit() {
```

Comment:
We should be careful here to not re-download mariner if we are already running interactive inside a mariner repo. We don't want to use the stable branch, we want to use our local copy.